doc/source/tutorial-quickstart-xgboost.rst

```diff
@@ -264,14 +265,15 @@ training, the last ``N=num_local_round`` trees will be extracted to send to the
 In ``evaluate``, after loading the global model, we call ``bst.eval_set`` function to
 conduct evaluation on valid set. The AUC value will be returned.
 
-The ServerApp
--------------
+ServerApp
```

Comment:
Same here

Reply:
Updated - also for Configuration and Data.